docs/source/deployorderer/ordererdeploy.md

```diff
@@ -105,13 +103,13 @@ At a minimum, you need to configure the following parameters:
 - `General.TLS.PrivateKey ` - Ordering node private key from TLS CA.
 - `General.TLS.Certificate ` - Ordering node signed certificate (public key) from the TLS CA.
 - `General.TLS.RootCAS` - This value should be unset.
-- `General.BoostrapMethod:none` - This allows the orderer to start without needing a system channel configuration block.
+- `General.BoostrapMethod:none` - This allows the orderer to start without needing a system channel configuration block (system channel is no longer supported since Fabric v3.0).
 - `General.LocalMSPDir` - Path to the ordering node MSP folder.
 - `General.LocalMSPID` - MSP ID of the ordering organization as specified in the channel configuration.
 - `FileLedger.Location` - Location on the file system to the ledgers of the channels this orderer will be servicing.
-- `ChannelParticipation.Enabled` - Set to `true`. This allows the orderer to be joined to an application channel without joining a system channel first.
+- `ChannelParticipation.Enabled` - Set to `true`. This allows the orderer to be joined to an application channel without joining a system channel first ("system channel" is no longer supported since Fabric v3.0).
```

Comment:
remove "without joining a system channel first", 
"system channel" -> the system channel